tools/launch_benchmark_suite_scheduler_slurm.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+import submitit
+from fvcore.common.file_io import PathManager
+from vissl.config.attr_dict import AttrDict
+from vissl.engines.benchmark_suite_scheduler import BenchmarkSuiteScheduler
+from vissl.utils.io import load_file
+
+
+# Default slurm options to pass to the executor.
+_DEFAULT_SLURM_OPTIONS = {
+    "NAME": "vissl",
+    "COMMENT": "vissl evaluation job",
+    "PARTITION": "learnlab",
+    "CONSTRAINT": "",
+    "TIMEOUT_MIN": 72 * 60,  # Timeout in minutes.
+    "CPUS_PER_TASK": 8,
+    "MEM_GB": 16,
+    "ADDITIONAL_PARAMETERS": {},
```

Comment:
nit: remove this.

Reply:
I like having this here, I think it gives some built-in documentation of what sort of options you can add to SLURM. Additional_parameters could be helpful for something like email notifcation setup.

Reply:
gotcha. this is something captured by the SLURM config already. Do we require a new description still? :)

Reply:
See https://github.com/facebookresearch/vissl/pull/325/files#r645636434

Reply:
I don't think I understand what you mean. We are not using the defaults.yaml here, we are providing our own default options here (mainly so that we can set the CPUS_PER_TASK and MEM_GB intelligently.

Reply:
TODO: Create defaults json file.